pkg/acceptance/bettertestspoc/assert/resourceshowoutputassert/task_show_output_gen.go

```diff
@@ -0,0 +1,158 @@
+	t.AddAssertion(assert.ResourceShowOutputValueSet("database_name", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasSchemaName(expected string) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputValueSet("schema_name", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasOwner(expected string) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputValueSet("owner", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasComment(expected string) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputValueSet("comment", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasWarehouse(expected string) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputStringUnderlyingValueSet("warehouse", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasSchedule(expected string) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputValueSet("schedule", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasState(expected sdk.TaskState) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputStringUnderlyingValueSet("state", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasDefinition(expected string) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputValueSet("definition", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasCondition(expected string) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputValueSet("condition", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasAllowOverlappingExecution(expected bool) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputBoolValueSet("allow_overlapping_execution", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasErrorIntegration(expected string) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputStringUnderlyingValueSet("error_integration", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasLastCommittedOn(expected string) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputValueSet("last_committed_on", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasLastSuspendedOn(expected string) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputValueSet("last_suspended_on", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasOwnerRoleType(expected string) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputValueSet("owner_role_type", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasConfig(expected string) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputValueSet("config", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasBudget(expected string) *TaskShowOutputAssert {
+	t.AddAssertion(assert.ResourceShowOutputValueSet("budget", expected))
+	return t
+}
+
+func (t *TaskShowOutputAssert) HasTaskRelations(expected sdk.TaskRelations) *TaskShowOutputAssert {
```

Comment:
is this method generated? if not, it should go to the ext file

Reply:
Moved to ext